### ISSUE TITLE
Repeating sets: Non-active element in tab order 

### ISSUE THREAD

**Issue by samsadasivan** [cds-snc/platform-forms-client#5839]
**_Specs: Windows
Screen Readers Used: JAWS, NVDA
Browsers Tested: Google Chrome, Microsoft Edge, Internet Explorer_**

**_Observation:_** 
The element has been placed in the Tab order using tabindex="0", but it lacks a specific role="..." attribute. 

The tabbable element does not have a valid widget role 

A tabbable element must have a valid widget role. 

 **_Severity: Critical_** 

**Recommendation:**  
Replace the non-widget role, and/or use the explicit tabindex="-1" attribute to remove the element from the keyboard focus (tab) order 

<img width="536" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMk9nQ2c9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--519d986671844eca377e7f68530670ecd92598d2/image.png" alt="image.png" />
